Dockerfile

```diff
@@ -32,6 +32,8 @@ RUN npx turbo prune --scope=@calcom/web --docker
 RUN yarn install
 RUN yarn db-deploy
 RUN yarn --cwd packages/prisma seed-app-store
+# Build and make embed servable from web/public/embed folder
+RUN yarn --cwd packages/embeds/embed-core workspace @calcom/embed-core run build
 RUN yarn --cwd apps/web workspace @calcom/web run build

```

Comment:
Note: Docker build was failing for me at @calcom/web run build command(due to type check) and it is unrelated to this change. I got the build passing through by adding CI=1 env variable for line 37.